ui/helper/Validators.js

```diff
@@ -15,6 +15,15 @@ const Validators = {
     },
     Boolean: {
         valid: b => typeof b === typeof true
+    },
+    URL: {
+        valid: u => {
+            try {
+                return !!new URL(u);
```

Comment:
Is it correct `! !`?

Reply:
That's correct, it's a quick way to parse any value into a `boolean`.